frontend/public/components/storage/shared.ts

```diff
@@ -32,25 +33,25 @@ export const initialAccessModes = ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMa
 export const accessModeRadios = [
   {
     value: 'ReadWriteOnce',
-    title: 'Single User (RWO)',
+    title: i18next.t('public~Single User (RWO)'),
```

Comment:
This won't be reactive. We should look for a different way to handle this instead of setting it once as a top-level constant in the module.